lib/WTSI/DNAP/Warehouse/Schema/Query/LibraryDigest.pm

```diff
@@ -0,0 +1,611 @@
+package WTSI::DNAP::Warehouse::Schema::Query::LibraryDigest;
+
+use Moose;
+use Carp;
+use Readonly;
+use MooseX::StrictConstructor;
+use List::MoreUtils qw/ none uniq /;
+
+our $VERSION = '0';
+
+Readonly::Hash   my %QUALITY_FILTERS => (
+                              'mqc'        => 'manual_qc',
+                              'extrelease' => 'external_release',
+                                        );
+
+Readonly::Array  my @AGGREGATYON_LEVEL => qw/
```

Comment:
Thanks, fixed.
>
> Typo: AGGREGATION not AGGREGATYON
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/wtsi-npg/ml_warehouse/pull/21/files#r24989056>.
>




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.